sdks/python/container/boot.go

```diff
@@ -157,15 +156,20 @@ func launchSDKProcess() error {
 	signalChannel := make(chan os.Signal, 1)
 	signal.Notify(signalChannel, syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM)
 
-	venvDir, err := setupVenv(ctx, logger, "/opt/apache/beam-venv", *id)
-	if err != nil {
-		return errors.New("failed to initialize Python venv")
-	}
-	cleanupFunc := func() {
-		os.RemoveAll(venvDir)
-		logger.Printf(ctx, "Cleaned up temporary venv for worker %v.", *id)
+	// Create a separate virtual envirionment (with access to globally installed packages), unless disabled by the user.
+	// This improves usability on runners that persit the execution environment for the boot entrypoint between multiple pipeline executions.
+	if os.Getenv("RUN_PYTHON_SDK_IN_DEFAULT_ENVIRONMENT") == "" {
+		venvDir, err := setupVenv(ctx, logger, "/opt/apache/beam-venv", *id)
+		if err != nil {
+			logger.Printf(ctx, "Using default environment, since creating a virtual environment for the SDK harness didn't succeed: "+err.Error())
```

Comment:
If my Go knowledge is correct, this will be still logged as INFO?

Reply:
As implemented, that's correct. The logger is just a lightweight wrapper to actually send it through the logging service instead of it hopefully getting picked up via STDOut + Err (with those being a fallback).  Info logs currently, since it doesn't seem valuable to differentiate at this time.

Reply:
Given this error is not critical, Info level is appropriate.